client/rest-high-level/src/test/java/org/elasticsearch/client/extension/CustomRestHighLevelClientTests.java

```diff
@@ -17,19 +17,17 @@
  * under the License.
  */
 
-package org.elasticsearch.client;
+package org.elasticsearch.client.extension;
```

Comment:
sorry I didn't notice this before, but why changing the package here? It would be nice to be able to test everything we need with reflection, rather than changing package... unless there's another reason why this was done.

Reply:
No other reason, just to make it more obvious that ` CustomRestClient` extends the high level rest client while being located in a different package.

Reply:
Please let's not make an arbitrary (and inconsistent) change like that. We do not use "extension" packages anywhere else.